src/pages/search/project_search/ProjectStatusFilter.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, MenuItem, TextField, Typography } from '@mui/material';
+import { ProjectSearchParameter } from '../../../api/cristinApi';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router';
+import { TFuncKey } from 'i18next';
+import React from 'react';
+
+interface FilterItem {
+  field: string;
+  i18nKey: TFuncKey;
+}
+
+enum Status {
+  ALL = 'ALL',
+  ACTIVE = 'ACTIVE',
+  CONCLUDED = 'CONCLUDED',
+  NOTSTARTED = 'NOTSTARTED',
+}
+
+export const ProjectStatusFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+
+  const onStatusChange = (event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
+    const status = event.target.value ?? '';
+    if (Object.values<string>(Status).includes(status) && status != Status.ALL) {
+      addStatusFilter(status);
+    } else {
+      removeStatusFilter();
+    }
+  };
+
+  const addStatusFilter = (status: string) => {
+    searchParams.set(ProjectSearchParameter.Status, status);
+    history.push({ search: searchParams.toString() });
+  };
+
+  const removeStatusFilter = () => {
+    searchParams.delete(ProjectSearchParameter.Status);
+    history.push({ search: searchParams.toString() });
+  };
+
+  const statusFilters: FilterItem[] = [
```

Comment:
denne er statisk, så den kan initieres over komponenten, slik at den ikke reinitialiseres for hver render